Mathlib/CategoryTheory/Grothendieck.lean

```diff
@@ -149,6 +153,108 @@ def forget : Grothendieck F ⥤ C where
 
 end
 
+section
+
+variable {G : C ⥤ Cat}
+
+/-- The Grothendieck construction is functorial: a natural transformation `α : F ⟶ G` induces
+a functor `Grothendieck.map : Grothendieck F ⥤ Grothendieck G`.
+-/
+@[simps!]
+def map (α : F ⟶ G) : Grothendieck F ⥤ Grothendieck G where
+  obj X := {
+    base := X.base
+    fiber := (α.app X.base).obj X.fiber
+  }
+  map {X Y} f := {
+    base := f.base
+    fiber := (eqToHom (α.naturality f.base).symm).app X.fiber ≫ (α.app Y.base).map f.fiber
+  }
+  map_id := by
+    intro X
+    dsimp
+    simp [eqToHom_map]
+    congr 1
+    · rw [eqToHom_app, eqToHom_trans]
+  map_comp := by
+    intro X Y Z f g
+    dsimp
+    congr 1
+    erw [comp_fiber f g]
```

Comment:
This `erw` suggests that there is a simp lemma missing. Maybe you can add a new version of `comp_fiber` after the `Category` instance phrased in terms of `≫` instead of `comp`?

Reply:
sounds a good idea!

Reply:
just added a new lemma `comp_fiber'` similar to already existing `id_fiber'` and now we use `simp [comp_fiber' f g]` instead of `erw [comp_fiber f g]`.